tasks/docker-compose.yml

```diff
@@ -1,20 +1,23 @@
+# vi: set ft=ansible :
 ---
 - name: Check current docker-compose version.
-  command: docker-compose --version
+  command: "{{ docker_compose_path }} --version"
```

Comment:
Can you break out this fix into a separate PR? This is a bug that should probably be fixed separately.